webapp/lib/controller.dart

```diff
@@ -272,36 +273,29 @@ void addsTagsChanged(String originalTag, String updatedTag, model.TagType tagTyp
 // Suggested Replies operations
 void addNewResponse() {
   model.packageConfigurationData[selectedPackage].suggestedReplies.add(
-    {
-      "messages":
-        [
-          "",
-          "",
-        ],
-      "reviewed": false,
-      "reviewed-by": "",
-      "reviewed-date": ""
-    },
-  );
+    new_model.SuggestedReply()
+    );
   loadPackageConfigurationView();
 }
 
 void updateResponse(int rowIndex, int colIndex, String response) {
-  model.packageConfigurationData[selectedPackage].suggestedReplies[rowIndex]['messages'][colIndex] = response;
+  model.packageConfigurationData[selectedPackage].suggestedReplies[rowIndex].text = response;
```

Comment:
This is doing a pretty strange thing at the moment - if you try to edit the translation, it will replace the text with the edited translation. The `colIndex` says whether you're editing the text or translation, but now using the model, we have named fields rather than indexes. I think the easiest would be to change this method to take `text` and `translation` instead of `colIndex` and `response` but requires then changes in the view to extract the data for both columns, and reduces the view to storing text + translation, rather than a number of alternative translations > 1.